### ISSUE TITLE
[not an issue] Tiled Upscale Script not working! & sdxl controlnet ?

### ISSUE THREAD

**Issue by dnl13** [jags111/efficiency-nodes-comfyui#7]
For me Tiled Upscale Script was not working, 
got it fixed by reimplementing https://github.com/BlenderNeko/ComfyUI_TiledKSampler

but still the ControlNet for SDXL is not working but searching for a fix. 
SD1.5 working fine. 

Also I have seen this issue https://github.com/jags111/efficiency-nodes-comfyui/issues/6 
which seems to be related with a missing new implementation of  https://github.com/BlenderNeko/ComfyUI_ADV_CLIP_emb

I am still trying to fix the Tiled Upscale Script sdxl controlnet model selection on my fork. 
Just wanted to let you know that i will send a push req if i get to work. 
Also feel free to help  

tl;dr 
sdxl refiner issue https://github.com/jags111/efficiency-nodes-comfyui/issues/6  fixed
Tiled Upscale Script works again but only controlnet sd 1.5 for now 

edited: 
btw i dont know if there is a controlnet tile sdxl model
maybe there is nothing to fix ?

**Comment by jags111** [comment]
awesome ..Just checking saw the problem with Tiled sampler issue and converging same. If you have spotted same . Please do send what revision helped solve the issue of same. Thanks .
Secondly on the Controlnet script is failing ass of now and working on fixing same.. Wonderful on finding the  solutions for same and will update from your fork also.

**Comment by dnl13** [comment]
Also, we should remove the restriction on choosing the controlnet model.
I have had much better results with the Controlnet Inpaint Model. 

Also, we can use the tiled upscaler script for sdxl if there is a tiling model. 
The sdxl inpaint model https://huggingface.co/diffusers/stable-diffusion-xl-1.0-inpainting-0.1 
unfortunately does not work, but other sdxl controlent models do 
```
class TSC_Tiled_Upscaler:
    @classmethod
    def INPUT_TYPES(cls):
        # Split the list based on the keyword "tile"
        cnet_filenames = [name for name in folder_paths.get_filename_list("controlnet")]

        return {"required": {"upscale_by": ("FLOAT", {"default": 1.25, "min": 0.01, "max": 8.0, "step": 0.05}),
                             "tile_size": ("INT", {"default": 512, "min": 256, "max": MAX_RESOLUTION, "step": 64}),
                             "tiling_strategy": (["random", "random strict", "padded", 'simple', 'none'],),
                             "tiling_steps": ("INT", {"default": 30, "min": 1, "max": 10000}),
                             "seed": ("INT", {"default": 0, "min": 0, "max": 0xffffffffffffffff}),
                             "denoise": ("FLOAT", {"default": .4, "min": 0.0, "max": 1.0, "step": 0.01}),
                             "use_controlnet": ("BOOLEAN", {"default": False}),
                             "tile_controlnet": (cnet_filenames,),
                             "strength": ("FLOAT", {"default": 1.0, "min": 0.0, "max": 10.0, "step": 0.01}),
                             },
                "optional": {"script": ("SCRIPT",)}}
       ```
